app/src/ui/multi-commit-operation/choose-branch/merge-choose-branch-dialog.tsx

```diff
@@ -29,11 +30,11 @@ export class MergeChooseBranchDialog extends BaseChooseBranchDialog {
 
     dispatcher.mergeBranch(
       repository,
-      selectedBranch.name,
+      selectedBranch,
       this.mergeStatus,
       operation === MultiCommitOperationKind.Squash
     )
-    this.props.onDismissed()
+    this.props.dispatcher.closePopup(PopupType.MultiCommitOperation)
```

Comment:
Why this change? 🤔

Reply:
It was not necessary, but it should have been that way to start with. A preventative measure from this logic interfering with another features logic.